test/Makefile

```diff
@@ -0,0 +1,12 @@
+provision = true
```

Comment:
We can do something like the following to parameterize this:
```
PROVISION?=true
```

Then it's easier to change it on the commandline, I think it should work something like `make run PROVISION=false`.

(referencing https://stackoverflow.com/questions/2826029/passing-additional-variables-from-command-line-to-make)

Reply:
Hi, 

It's working like that:

```
$ git:(GinkgoMakefile) ✗ make run provision=false  
ginkgo --focus " Runtime*" -v -- --cilium.provision=false  
$ git:(GinkgoMakefile) ✗ make run provision=true 
ginkgo --focus " Runtime*" -v -- --cilium.provision=true   
```

Reply:
Ah, I didn't realise that Makefiles will not overwrite variables passed like that. OK :)